kirega/iReporter
# Ft create incident api #162181616

#### What does this PR do?
It creates the necessary endpoints for creating an endpoint
`POST /api/v1/incidents`
#### Description of Task to be completed?
- Creates the incidents model
- writes the necessary tests
#### How should this be manually tested?
Clone the repo.
`cd iReporter`
`pip install -r requirements.txt`
`flask run`
using postman or other appropriate endpoints tester,
post  the payload:
`{
            "comment": "Police taking a bribe",
            "incidentType": "red-flag",
            "location": "-1.28333, 36.81667",
            "createdBy": 1,
            "images": [image_in_base64, image_in_base64],
            "videos": ["http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ElephantsDream.mp4",
                       "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerFun.mp4"]
        }`
respose expected is `{"message": "New incident created", "data": new_incident}`
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162181682
